submit.php

```diff
@@ -227,9 +227,13 @@
     <p>
     <?php
       if ($query && $mail_result) {
-        print "Your PTO has been put into the database, and the email has been sent :)";
+        print "Your PTO has been put into the database, and the email has been sent.\n";
+        print "Please remember to <b>BLOCK OFF THIS TIME IN YOUR CALENDAR</b> and mark yourself as <b>BUSY</b>.\n";
```

Comment:
Two things: `\n` doesn't do anything useful in HTML. Also, since you're repeating yourself you might want to use a variable or pull it outside of this if block somehow.